CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Layout Framework Version History
 
+**Changes in version 1.10**
+- Fixed gaps between fields in v18 layout.
```

Comment:
Can you please be more specific. You only modify one layout, and only a few fields, so you can list them. Something like this:
```
Fixed wrong length for the following fields: field1, field2...
```